include/openthread/border_routing.h

```diff
@@ -131,6 +131,20 @@ typedef struct otBorderRoutingPrefixTableEntry
     uint32_t                   mPreferredLifetime;   ///< Preferred lifetime of the on-link prefix when `mIsOnLink`.
 } otBorderRoutingPrefixTableEntry;
 
+/**
+ * Represents an entry from the discovered NAT64 prefix table.
+ *
+ * The entries in the discovered table track the Nat64 Prefix Options in the received Router Advertisement messages from
+ * other routers on the infrastructure link.
+ */
+typedef struct otBorderRoutingNat64PrefixEntry
+{
+    otBorderRoutingRouterEntry mRouter;              ///< Information about the router advertising this Nat64 prefix.
+    otIp6Prefix                mPrefix;              ///< The discovered IPv6 prefix.
+    uint32_t                   mMsecSinceLastUpdate; ///< Milliseconds since last update of this prefix.
+    uint32_t                   mLifetime;            ///< Lifetime of the prefix (in seconds).
```

Comment:
Is this the initially advertised lifetime, or currently remaining lifetime? Maybe the comment could indicate this.